orix/quaternion/quaternion.py

```diff
@@ -498,3 +499,72 @@ def _outer_dask(
 
         new_chunks = tuple(chunks1[:-1]) + tuple(chunks2[:-1]) + (-1,)
         return out.rechunk(new_chunks)
+
+    @classmethod
+    def from_scipy_rotation(cls, rotation: SciPyRotation) -> Quaternion:
+        """Return quaternion(s) from :class:`scipy.spatial.transform.Rotation`(s).
+
+        Parameters
+        ----------
+        rotation
+            rotation of type :class:`scipy.spatial.transform.Rotation.
```

Comment:
No need to repeat the class type when it's already linked to in the type hints (see [here](https://orix--401.org.readthedocs.build/en/401/reference/generated/orix.quaternion.Quaternion.from_scipy_rotation.html)).

```suggestion
            SciPy rotation(s).
```